Library/Homebrew/extend/ARGV.rb

```diff
@@ -268,6 +268,10 @@ def fetch_head?
     include? "--fetch-HEAD"
   end
 
+  def ignore_installed_specs?
+    include? "--ignore-installed-specs"
```

Comment:
Hmm. I guess I'm still not unsure when you would want to pass this option. I'd consider making this PR without it. CC @ilovezfs for thoughts.